R/constructors.R

```diff
@@ -141,6 +142,13 @@ new_quant_param <- function(type = c("double", "integer"),
     }
   }
 
+  # if (!is_unknown(default)) {
+  #   lifecycle::deprecate_warn(
+  #     when = "1.0.0",
+  #     what = "new_quant_param(default)",
+  #     details = "It will be removed in a future release."
+  #   )
+  # }
```

Comment:
Make sure that you don't include `default` in the returned object anymore either